lms/djangoapps/mailing/management/commands/mailchimp_id.py

```diff
@@ -3,9 +3,8 @@
 a valid list.
 """
 import sys
-from optparse import make_option
 
-from django.core.management.base import BaseCommand, CommandError
+from django.core.management.base import BaseCommand
 from mailsnake import MailSnake
```

Comment:
🐍 ? 🐵 ? crazy...

Reply:
Mail can take many forms, it's best not to dwell on it too much.